contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM resin/rpi-raspbian:jessie
```

Comment:
The build of the `resin/rpi-raspbian` Docker images is quite clean, small and well documented in this repo https://github.com/resin-io-library/resin-rpi-raspbian. I'd like to suggest to use these ones. The builds can be run on an Intel build server and therefore could be used for official images, too.